src/component/axis/AxisBuilder.ts

```diff
@@ -17,17 +17,17 @@
 * under the License.
 */
 
-import {retrieve, defaults, extend, each, isObject, map} from 'zrender/src/core/util';
+import { retrieve, defaults, extend, each, isObject, map } from 'zrender/src/core/util';
```

Comment:
Please avoid bringing the code style change in your pull request. It makes us harder to review the real functional change from your PR.